fbw-a32nx/src/systems/instruments/src/PFD/AltitudeIndicator.tsx

```diff
@@ -326,18 +337,10 @@ export class AltitudeIndicatorOfftape extends DisplayComponent<AltitudeIndicator
         this.abnormal.instance.style.display = 'none';
         this.normal.instance.removeAttribute('style');
       }
+      this.altFlagVisible.set(!altitude.isNormalOperation());
     });
 
-    sub
-      .on('tcasFail')
-      .whenChanged()
-      .handle((tcasFailed) => {
-        if (tcasFailed) {
-          this.tcasFailed.instance.style.display = 'inline';
-        } else {
-          this.tcasFailed.instance.style.display = 'none';
-        }
-      });
+    this.tcasFailed.setConsumer(sub.on('tcasFail').whenChanged());
```

Comment:
```suggestion
    this.tcasFailed.setConsumer(sub.on('tcasFail'));
```
The whenChanged is redundant, as the ConsumerSubject already does an equality test.